barretenberg/cpp/src/barretenberg/vm2/simulation_helper.cpp

```diff
@@ -459,6 +466,30 @@ TxSimulationResult AvmSimulationHelper::simulate_fast_with_hinted_dbs(const Exec
     return simulate_fast(raw_contract_db, raw_merkle_db, hints.tx, hints.globalVariables, hints.protocolContracts);
 }
 
+// Note: we currently only have hinted raw dbs, TODO eventually remove hints:
+TxSimulationResult AvmSimulationHelper::simulate_fast_without_hinted_dbs(const ExecutionHints& hints)
```

Comment:
Given that there are so many hints this comment is a bit ambiguous. Maybe call it `fast_with_real_DBs` and in the TODO say `use real DBs`? because you still want to collect hints, I guess.